doc/Internals.md

```diff
@@ -88,10 +88,6 @@ associated with a project, except the root. The root can hold multiple directori
 each representing a different project. This is why the root will have an empty `proj\_id`,
 and an empty `proj\_folder`.
 
-The local directory contents does not change after the describe calls
```

Comment:
I removed it because i dont think it's true. We do update the `directories` table when creating new folder and removing empty one. @kpjensen